registry/coder-labs/modules/auggie/README.md

```diff
@@ -0,0 +1,190 @@
+---
+display_name: Auggie CLI
+icon: ../../../../.icons/auggie.svg
+description: Run Auggie CLI in your workspace for AI-powered coding assistance with AgentAPI integration
+verified: true
+tags: [agent, auggie, ai, tasks, augment]
+---
+
+# Auggie CLI
+
+Run Auggie CLI in your workspace to access Augment's AI coding assistant with advanced context understanding and codebase integration. This module integrates with [AgentAPI](https://github.com/coder/agentapi).
+
+```tf
+module "auggie" {
+  source   = "registry.coder.com/coder-labs/auggie/coder"
+  version  = "0.1.0"
+  agent_id = coder_agent.example.id
+  folder   = "/home/coder/project"
+}
+```
+
+## Prerequisites
+
+- You must add the [Coder Login](https://registry.coder.com/modules/coder/coder-login) module to your template
+- **Augment session token for authentication (required for tasks)**
```

Comment:
Can we add a link to the docs or the AugmentCode dashboard on how to get the token?